HW2/P3/mandelbrot.pyx

```diff
@@ -69,9 +134,24 @@ cpdef example_sqrt_8(np.float32_t[:] values):
     # mask will be true where 2.0 < avxval
     mask = AVX.less_than(AVX.float_to_float8(2.0), avxval)
 
+    mask2 = AVX.less_than(AVX.float_to_float8(2.0), AVX.float_to_float8(1.0))
+    mask3 = AVX.greater_than(AVX.float_to_float8(2.0), AVX.float_to_float8(1.0))
+
+    print_complex_AVX(mask2, mask3)
+
+    mask4 = AVX.bitwise_andnot(mask2, mask3)
+
+    print_complex_AVX(mask4, AVX.bitwise_and(mask4, AVX.float_to_float8(5.0)))
+
+    sign = AVX.signs(AVX.float_to_float8(-1))
+    sign2 = AVX.signs(AVX.float_to_float8(0))
+    sign3 = AVX.signs(AVX.float_to_float8(1))
+    sign4 = AVX.signs(AVX.make_float8(1,2,3,4,-1,-2,-3,-4))
+
```

Comment:
There is a easier way to do this problem. Please add some comments so I could follow